setup.py

```diff
@@ -82,12 +83,25 @@ def package_version():
         package_manifest = json.load(package_file)
         return package_manifest['version']
 
+supported_frameworks = [
```

Comment:
Remove.

Reply:
Currently no strong compliance. Either way such formatting changes should be discussed and done consistently across files, not rolled into other PRs.

Reply:
@lutzroeder I have removed new formatting

is the discussion resolved now?

Reply:
Can you also remove the addition in `long_description`. This is not as complete as support for other frameworks and would also like to do some testing and see some user testing before adding it.

Reply:
No problem, I have removed it. Please list what should be done in my next PR to make it possible for the OpenVINO to appear in the list of experimentally supported frameworks.